test/val/val_layout_test.cpp

```diff
@@ -136,53 +126,54 @@ INSTANTIATE_TEST_CASE_P(InstructionsOrder,
     // are not "All" in the below.
     //
     //                                  | Instruction                | Line(s) valid          | Lines to compile
```

Comment:
The headers for this should should be realigned.

Reply:
done